### PR TITLE
Fix Module Stuff

### PR DESCRIPTION
**Pull request by 12944qwerty**

### CODE REVIEW

**Review by asportnoy — CHANGES_REQUESTED** [review]
The docs are missing any form of usable information for all common modules and all components except for ButtonItem, Category, and Tooltip.

**Review by asportnoy — CHANGES_REQUESTED** [review]
https://github.com/replugged-org/replugged/pull/294#discussion_r1055930726
https://github.com/replugged-org/replugged/pull/294#discussion_r1056103939
Both are still not fixed

File: src/renderer/modules/webpack/common/modal.ts [hunk 1]
```diff
@@ -52,32 +52,32 @@ export interface ModalComponents {
...
-  Direction: string;
-  Align: string;
-  Justify: string;
-  Wrap: string;
-};
+  ModalSize: typeof ModalSize;
+  Direction: Record<string, string>;
+  Align: Record<string, string>;
+  Justify: Record<string, string>;
+  Wrap: Record<string, string>;
```
Comment (asportnoy) [inline src/renderer/modules/webpack/common/modal.ts:66]: Can we be more specific with the keys here

**Review by asportnoy — CHANGES_REQUESTED** [review]

File: src/renderer/modules/webpack/common/modal.ts (hunk 1 above)
Comment (12944qwerty) [reply src/renderer/modules/webpack/common/modal.ts:66]: Those are the key names within discord and throughout client modding lol. They give the classes for the modal.

Comment (asportnoy) [reply src/renderer/modules/webpack/common/modal.ts:66]: @12944qwerty but what are the keys?

Comment (12944qwerty) [reply src/renderer/modules/webpack/common/modal.ts:66]: Ohhh, you want to limit the keynames... gotcha.

**Review by asportnoy — APPROVED** [review]

**Review by FifiTheBulldog — APPROVED** [review]

**Pull request merged**